test/test_database.py

```diff
@@ -138,29 +138,133 @@ def db(self, dependency_provider):
     @patch.object(Session, 'rollback')
     @patch.object(Session, 'commit')
     @patch.object(Session, 'close')
-    def test_comits_and_closes(self, close, commit, rollback, db):
+    def test_comits(self, close, commit, rollback, db):
 
         with db.get_session() as session:
             assert isinstance(session, Session)
 
+        commit.assert_called()
+        rollback.assert_not_called()
```

Comment:
`assert commit.called` and `assert not rollback.called` are _slightly_ less likely to get you in trouble with a typo (e.g. `commit.asssert_called()` will always be true and doesn't come with any syntax highlighting)

Reply:
true, one word less for making a typo .) Plus the syntax highlight, plus making it nicely stand in line with the other standard assertions.